universal-application-tool-0.0.1/build.sbt

```diff
@@ -100,12 +105,14 @@ JsEngineKeys.engineType := JsEngineKeys.EngineType.Node
 resolvers += Resolver.bintrayRepo("webjars","maven")
 libraryDependencies ++= Seq(
     "org.webjars.npm" % "react" % "15.4.0",
-    "org.webjars.npm" % "types__react" % "15.0.34"
+    "org.webjars.npm" % "types__react" % "15.0.34",
+    "org.webjars.npm" % "azure__storage-blob" % "10.5.0",
+    "org.webjars" % "requirejs" % "2.2.0",
```

Comment:
question (blocking): is requirejs needed for this PR?

Reply:
Nope sorry, added it when testing and forgot to remove.